ml-agents/mlagents/envs/utilities.py

```diff
@@ -1,19 +1,94 @@
 from PIL import Image
 import numpy as np
 import io
+import logging
 
+from mlagents.envs.brain import BrainParameters, BrainInfo
 
-def process_pixels(image_bytes, gray_scale):
+logger = logging.getLogger("mlagents.envs")
+
+
+def process_pixels(image_bytes, grey_scale):
```

Comment:
If `process_pixels` is only used in `brain_param_proto_to_obj` and we move `brain_param_proto_to_obj` inside of BrainInfo maybe it should not be in `utilities.py` anymore?